.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
The convention in this project is to not insert a final newline. Otherwise the settings look good to me.

Reply:
No! There are final newlines throughout the project!

`lib/timezone.rb` at your `master`:

![image](https://user-images.githubusercontent.com/6510020/33165736-ef500cac-d048-11e7-83af-10e237213c26.png)

[And it's good practice for UNIX](https://stackoverflow.com/a/729795/2630849) (try to `cat lib/timezone.rb` with newline and without).

GitHub will display files without final newlines abnormally, with the crossed out symbol like this `↲` ("Enter") at the last line.

Reply:
Is this setting adding an additional blank line or just a newline on line 79?

Reply:
> Is this setting adding an additional blank line or just a newline on line 79?

These are equivalent things :grimacing: 

Adding newline at 79 makes blank (empty) 80 line.

Try to `$ cat lib/timezone.rb` with newline (and 80 line, which doesn't show in GitHub, for example) and without these things.